worker/task/process_event_creation.go

```diff
@@ -154,6 +155,35 @@ func ProcessEventCreated(appRepo datastore.ApplicationRepository, eventRepo data
 			}
 		}
 
+		// convert event data field to map
```

Comment:
@jirevwe I think we should create another queue called `index-events` and create a separate task for it. It will be a low-priority queue that continues to index events from time to time. I think we will achieve better isolation to test that routine and take this out of our critical path of event delivery. What do you think? 

My first option was to see if `asynq` allows multiple handlers to read off the same queue, but It seems impossible.

Reply:
Sure

Reply:
I have modified this